src/main/java/com/mide/gangsaeng/bannedword/BannedWordServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.mide.gangsaeng.bannedword;
+
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class BannedWordServiceImpl implements BannedWordService {
+
+    private static ConcurrentHashMap<String,BannedWord> cache;
```

Comment:
1.  If we use `HashMap`
    - thread A - `cache.put("bitch", new BannedWord("bitch")`
    - thread B - `cache.remove("bitch", new BannedWord("bitch")`
  
  If one thread tries to remove a specific key while another thread simultaneously accesses the same key to add a value, it may result in a `ConcurrentModificationException` or leave unexpected values in the map.

2. For now, `Set` looks fine, but considering scalability and modification, I used `HashMap`. It is more useful in modification than `Set`.